FuncGameDev/FSharp/HighScore.txt

```diff
@@ -1 +1,3 @@
-0
\ No newline at end of file
+0
+0
+0
```

Comment:
Concerned by this. Won't this mean that we never save a "lowest remaining time," since it will never be less than 0?